Makefile

```diff
@@ -26,26 +31,51 @@ pkg/generated/openapi/%/zz_generated.openapi.go: go.mod go.sum
 	    -o ./ \
 	    -r /dev/null
 
+
+# Build
+# -----
+
 .PHONY: build-test-adapter
 build-test-adapter: $(generated_files)
 	CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) go build -o $(OUT_DIR)/$(ARCH)/test-adapter sigs.k8s.io/custom-metrics-apiserver/test-adapter
 
-.PHONY: gofmt
-gofmt:
-	./hack/gofmt-all.sh
+
+# Format and lint
+# ---------------
+
+GOLANGCI_VERSION:=1.50.1
```

Comment:
nit: this would be more convenient to have at the top-level for the future since it would allow having all the versions of the tools defined in one block